mozilla-release/config/version_win.pl

```diff
@@ -113,7 +113,7 @@ sub getNextEntry
 
 my $bufferstr="    ";
 
-my $MILESTONE_FILE = "$topsrcdir/config/milestone.txt";
+my $MILESTONE_FILE = "$topsrcdir/browser/config/version_display.txt";
```

Comment:
keeping $MILESTONE_FILE variable is pretty misleading. Cannot we rename it to $VERSION_DISPLAY_FILE?